my_api/search_sync.py

```diff
@@ -60,8 +66,7 @@ async def run_main(self):
 
             posts_to_index = self.serialize_posts(posts_past_time)
             print("posts:\n", posts_to_index)
-            self.client.index(SEARCH_INDEX_NAME).add_documents(posts_to_index)
-
+            await self.client.index(SEARCH_INDEX_NAME).add_documents(posts_to_index)
 
-    def serialize_posts(self, posts: list[Post]) -> list[dict[str, any]]:
+    def serialize_posts(self, posts: list[Post]) -> list[dict[str, Any]]:
```

Comment:
`any` isn't a valid type, it should be `Any` (and needs to be imported)

Reply:
Thank you for reaching out and making these changes! I am a pretty green developer trying to learn how to use Meilisearch and the packages you have worked on look like they'll help a lot (the sdk and pytest built for meilisearch).

Reply:
Feel free to reach out on either repo if you have questions.